static/app/views/settings/account/apiNewToken.tsx

```diff
@@ -61,34 +67,46 @@ export default class ApiNewToken extends Component<{}, State> {
               }
             )}
           </TextBlock>
-          <Panel>
-            <PanelHeader>{t('Permissions')}</PanelHeader>
-            <ApiForm
-              apiMethod="POST"
-              apiEndpoint="/api-tokens/"
-              initialData={{scopes: []}}
-              onSubmitSuccess={this.onSubmitSuccess}
-              onCancel={this.onCancel}
-              footerStyle={{
-                marginTop: 0,
-                paddingRight: 20,
-              }}
-              submitDisabled={Object.values(permissions).every(
-                value => value === 'no-access'
-              )}
-              submitLabel={t('Create Token')}
-            >
-              <PanelBody>
-                <PermissionSelection
-                  appPublished={false}
-                  permissions={permissions}
-                  onChange={value => {
-                    this.setState({permissions: value});
-                  }}
-                />
-              </PanelBody>
-            </ApiForm>
-          </Panel>
+          {newToken ? (
```

Comment:
explicit null check is preferred
```suggestion
          {newToken === null ? (
```